protocols/v2/roles-logic-sv2/src/channel_logic/channel_factory.rs

```diff
@@ -43,7 +43,7 @@ pub struct PartialSetCustomMiningJob {
 pub enum OnNewShare {
     /// Used when the received is malformed, is for an inexistent channel or do not meet downstream
     /// target.
-    SendErrorDowsntream(SubmitSharesError<'static>),
+    SendErrorDownstream(SubmitSharesError<'static>),
```

Comment:
On line 1481 in this file, SendErrorDowsntream needs to be updated (tests don't compile).